Assets/MixedRealityToolkit/MSBuild/MSBuildTemplates/SolutionTemplate.sln.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
I notice you have a `.meta` file for `SolutionTemplate.sln` and `SDKProjectTemplate.csproj`. These files don't seem to be checked in on this PR's branch. Also, when I synced your fork and tried to run the _Generate C# SDK Projects_ command from Unity, I got the following errors:
![image](https://user-images.githubusercontent.com/47415945/60821368-52a41a80-a19b-11e9-8feb-c48fb21808df.png)
Did you forget to commit these files?

Reply:
Dang, yeah, these files got git ignored. Will re-add, thanks!

Reply:
Yep, added them

---
In reply to: [301149984](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5215#discussion_r301149984) [](ancestors = 301149984)